undertow/src/main/java/org/wildfly/extension/undertow/filters/Filter.java

```diff
@@ -54,7 +55,7 @@ public void registerOperations(ManagementResourceRegistration resourceRegistrati
         FilterAdd add = new FilterAdd(this);
         registerAddOperation(resourceRegistration, add, OperationEntry.Flag.RESTART_RESOURCE_SERVICES);
         //registerRemoveOperation(resourceRegistration, new ServiceRemoveStepHandler(UndertowService.FILTER, add), OperationEntry.Flag.RESTART_RESOURCE_SERVICES);
-        registerRemoveOperation(resourceRegistration, ReloadRequiredRemoveStepHandler.INSTANCE, OperationEntry.Flag.RESTART_RESOURCE_SERVICES);
+        registerRemoveOperation(resourceRegistration, new ServiceRemoveStepHandler(UndertowService.FILTER, add), OperationEntry.Flag.RESTART_RESOURCE_SERVICES);
```

Comment:
This new line exactly matches what is commented out above.

Reply:
I did not notice that. I have no idea why it was commented out, it was added commented out in the initial implementation. I think it is just an oversight as there is no reason why these services cannot be removed.